ovn-fake-multinode-utils/get-config-value.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
`ovn-fake-multinode-utils/translate_yaml.py` was recently added.  We should change python to python3 there too I guess.

Reply:
Done